requirements.txt

```diff
@@ -69,3 +69,4 @@ django-debug-toolbar==3.1.1
 django-debug-toolbar-request-history==0.1.3
 vcrpy==4.1.1
 vcrpy-unittest==0.1.7
+pyjwt==1.7.1
```

Comment:
The real name of the package is `PyJWT` could we keep the same case?

Reply:
Verified it isn't an issue to keep the case. Committed the change.